2016/09/23 14:08:29 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "contact".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(270): CController->missingAction('contact')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('contact')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii_products/index.php/site/contact
HTTP_REFERER=http://localhost/yii_products/index.php/site/page?view=about
---
2016/09/23 14:51:54 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.product" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:323
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('index')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/gii/crud/index
HTTP_REFERER=http://localhost/yii_products/index.php/gii/crud/index
---
2016/09/23 14:52:32 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.Product" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:323
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('index')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/gii/crud/index
HTTP_REFERER=http://localhost/yii_products/index.php/gii/crud/index
---
2016/09/23 14:52:59 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.product.Product" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:323
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('index')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/gii/crud/index
HTTP_REFERER=http://localhost/yii_products/index.php/gii/crud/index
---
2016/09/23 15:02:35 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.Product.models" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:323
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('index')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/gii/crud/index
HTTP_REFERER=http://localhost/yii_products/index.php/gii/crud/index
---
2016/09/23 15:03:23 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.Product.Product" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:323
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('index')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/gii/crud/index
HTTP_REFERER=http://localhost/yii_products/index.php/gii/crud/index
---
2016/09/23 15:04:36 [error] [exception.CException] exception 'CException' with message 'Alias "application.modules.Product.Product" is invalid. Make sure it points to an existing PHP file and the file is readable.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:323
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\generators\crud\CrudCode.php(59): YiiBase::import('application.mod...', true)
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): CrudCode->validateModel('model', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(CrudCode), 'model')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CValidator->validate(Object(CrudCode), NULL)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(159): CModel->validate()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('index')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/gii/crud/index
HTTP_REFERER=http://localhost/yii_products/index.php/gii/crud/index
---
2016/09/23 15:08:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "product".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('product')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/product
---
2016/09/23 15:08:56 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('admin')
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii_products/index.php/Product/product/admin
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 15:10:20 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('admin')
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii_products/index.php/Product/product/admin
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/index
---
2016/09/23 15:24:58 [error] [php] include(User.php): failed to open stream: No such file or directory (D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:432)
Stack trace:
#0 D:\xampp\htdocs\yii_products\protected\components\UserIdentity.php(20): spl_autoload_call()
#1 D:\xampp\htdocs\yii_products\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(159): CInlineValidator->validate()
#5 D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php(56): LoginForm->validate()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): SiteController->runAction()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): SiteController->run()
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CWebApplication->run()
REQUEST_URI=/yii_products/index.php/site/login
in D:\xampp\htdocs\yii_products\protected\components\UserIdentity.php (20)
in D:\xampp\htdocs\yii_products\protected\models\LoginForm.php (52)
in D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php (56)
2016/09/23 15:39:32 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('admin')
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii_products/index.php/Product/product/admin
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 16:09:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/admin".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/admin')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/admin
---
2016/09/23 16:19:51 [error] [php] include(Product.php): failed to open stream: No such file or directory (D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:432)
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\db\ar\CActiveRecord.php(395): spl_autoload_call()
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CActiveDataProvider.php(165): model()
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel()
#3 D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php(27): CActiveDataProvider->__construct()
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): SiteController->runAction()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): SiteController->run()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\xampp\htdocs\yii_products\index.php(13): CWebApplication->run()
REQUEST_URI=/yii_products/index.php/site/index
in D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php (27)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 16:20:12 [error] [php] include(Product.php): failed to open stream: No such file or directory (D:\xampp\htdocs\yii-1.1.17.467ff50\framework\YiiBase.php:432)
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\db\ar\CActiveRecord.php(395): spl_autoload_call()
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CActiveDataProvider.php(165): model()
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CActiveDataProvider.php(87): CActiveDataProvider->getModel()
#3 D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php(27): CActiveDataProvider->__construct()
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): SiteController->runAction()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): SiteController->run()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#11 D:\xampp\htdocs\yii_products\index.php(13): CWebApplication->run()
REQUEST_URI=/yii_products/index.php/site/index
in D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php (27)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 16:33:11 [error] [php] Undefined variable: dataProvider (D:\xampp\htdocs\yii_products\protected\views\site\index.php:17)
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(869): SiteController->renderFile()
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(782): SiteController->renderPartial()
#2 D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php(27): SiteController->render()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(286): SiteController->runAction()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): SiteController->run()
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#10 D:\xampp\htdocs\yii_products\index.php(13): CWebApplication->run()
REQUEST_URI=/yii_products/index.php
in D:\xampp\htdocs\yii_products\protected\views\site\index.php (17)
in D:\xampp\htdocs\yii_products\protected\controllers\SiteController.php (27)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 16:57:13 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 16:57:13 [error] [exception.CException] exception 'CException' with message 'Property "Product.imgFileName" is not defined.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CComponent.php:173
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\db\ar\CActiveRecord.php(161): CComponent->__set('imgFileName', Object(CUploadedFile))
#1 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(69): CActiveRecord->__set('imgFileName', Object(CUploadedFile))
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii_products/index.php/Product/product/create
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 16:59:30 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 16:59:30 [error] [exception.CException] exception 'CException' with message 'Property "Product.image" is not defined.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\db\ar\CActiveRecord.php(145): CComponent->__get('image')
#1 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(71): CActiveRecord->__get('image')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii_products/index.php/Product/product/create
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 17:00:30 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:00:30 [error] [exception.CException] exception 'CException' with message 'Property "Product.image" is not defined.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\db\ar\CActiveRecord.php(145): CComponent->__get('image')
#1 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(71): CActiveRecord->__get('image')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii_products/index.php/Product/product/create
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 17:02:02 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:02:02 [error] [php] move_uploaded_file(): The second argument to copy() function cannot be a directory (D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CUploadedFile.php:201)
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): ProductController->runAction()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): ProductController->filterAccessControl()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): ProductController->runActionWithFilters()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): ProductController->run()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 D:\xampp\htdocs\yii_products\index.php(13): CWebApplication->run()
REQUEST_URI=/yii_products/index.php/Product/product/create
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (71)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:09:45 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:14:39 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:14:57 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:15:03 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:15:03 [error] [php] move_uploaded_file(): The second argument to copy() function cannot be a directory (D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CUploadedFile.php:201)
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): ProductController->runAction()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): ProductController->filterAccessControl()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): ProductController->runActionWithFilters()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): ProductController->run()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#12 D:\xampp\htdocs\yii_products\index.php(13): CWebApplication->run()
REQUEST_URI=/yii_products/index.php/Product/product/create
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (71)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:17:25 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:17:25 [error] [exception.CException] exception 'CException' with message 'Property "ProductController._tempName" is not defined.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(71): CComponent->__get('_tempName')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('create')
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#13 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/yii_products/index.php/Product/product/create
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 17:19:56 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:19:56 [error] [exception.CException] exception 'CException' with message 'Property "Product.image" is not defined.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CComponent.php:130
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\db\ar\CActiveRecord.php(145): CComponent->__get('image')
#1 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(72): CActiveRecord->__get('image')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#14 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii_products/index.php/Product/product/create
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 17:20:19 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:23:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:24:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:24:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
---
2016/09/23 17:25:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:25:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
---
2016/09/23 17:26:01 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:27:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:28:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:28:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:28:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:28:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:33:40 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php:158
Stack trace:
#0 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(51): ProductController->loadModel('10')
#1 [internal function]: ProductController->actionView('10')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(ProductController), Array)
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ProductController), Object(ReflectionMethod), Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('view')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#14 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii_products/index.php/Product/product/view/id/10
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 17:34:10 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:38:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php:159
Stack trace:
#0 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(51): ProductController->loadModel('11')
#1 [internal function]: ProductController->actionView('11')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(ProductController), Array)
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ProductController), Object(ReflectionMethod), Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('view')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#14 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii_products/index.php/Product/product/view/id/11
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/create
---
2016/09/23 17:39:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested page does not exist.' in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php:159
Stack trace:
#0 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(51): ProductController->loadModel('11')
#1 [internal function]: ProductController->actionView('11')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CAction.php(109): ReflectionMethod->invokeArgs(Object(ProductController), Array)
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(ProductController), Object(ReflectionMethod), Array)
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('view')
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#14 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii_products/index.php/Product/product/view/id/11
---
2016/09/23 17:39:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php:483
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(270): CController->missingAction('images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('images')
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#4 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii_products/index.php/Product/product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/
---
2016/09/23 17:39:07 [error] [exception.CException] exception 'CException' with message 'Property "CFileValidator.0" is not defined.' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CComponent.php:173
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\validators\CValidator.php(177): CComponent->__set('0', 'safe')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(287): CValidator::createValidator('file', Object(Product), 'imgFileName', Array)
#2 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(260): CModel->createValidators()
#3 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CModel.php(304): CModel->getValidators('title')
#4 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\helpers\CHtml.php(1514): CModel->isAttributeRequired('title')
#5 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\widgets\CActiveForm.php(600): CHtml::activeLabelEx(Object(Product), 'title', Array)
#6 D:\xampp\htdocs\yii_products\protected\modules\Product\views\product\_form.php(24): CActiveForm->labelEx(Object(Product), 'title')
#7 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CBaseController.php(126): require('D:\\xampp\\htdocs...')
#8 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\xampp\\htdocs...', Array, true)
#9 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(869): CBaseController->renderFile('D:\\xampp\\htdocs...', Array, true)
#10 D:\xampp\htdocs\yii_products\protected\modules\Product\views\product\create.php(18): CController->renderPartial('_form', Array)
#11 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CBaseController.php(126): require('D:\\xampp\\htdocs...')
#12 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CBaseController.php(95): CBaseController->renderInternal('D:\\xampp\\htdocs...', Array, true)
#13 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(869): CBaseController->renderFile('D:\\xampp\\htdocs...', Array, true)
#14 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#15 D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php(81): CController->render('create', Array)
#16 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\actions\CInlineAction.php(49): ProductController->actionCreate()
#17 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#19 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilter.php(40): CFilterChain->run()
#20 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#21 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#22 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#23 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(291): CFilterChain->run()
#24 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#25 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(282): CController->run('create')
#26 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/product...')
#27 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#28 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#29 {main}
REQUEST_URI=/yii_products/index.php/Product/product/create
HTTP_REFERER=http://localhost/yii_products/index.php/Product/product/
---
2016/09/23 17:40:00 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:41:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:41:42 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:41:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:41:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images/images.jpg".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images/...')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/images.jpg
HTTP_REFERER=http://localhost/yii_products/index.php/Product/Product
---
2016/09/23 17:41:49 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "Product/images/images.jpg".' in D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\web\CWebApplication.php(141): CWebApplication->runController('Product/images/...')
#1 D:\xampp\htdocs\yii-1.1.17.467ff50\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 D:\xampp\htdocs\yii_products\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii_products/index.php/Product/images/images.jpg
---
2016/09/23 17:46:32 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
2016/09/23 17:47:26 [warning] [application] Failed to set unsafe attribute "imgFileName" of "Product".
in D:\xampp\htdocs\yii_products\protected\modules\Product\controllers\ProductController.php (68)
in D:\xampp\htdocs\yii_products\index.php (13)
